ts/src/test/base/test.afterConstructor.ts

```diff
@@ -5,15 +5,13 @@ import assert from 'assert';
 import ccxt from '../../../ccxt.js';
 import testSharedMethods from '../Exchange/base/test.sharedMethods.js';
 
-function testAfterConstructor () {
-
+function helperTestInitThrottler () {
```

Comment:
in C#, if function starts with `test` , the C# bootstraper tests all those methods, so we need to prefix "in-file helper methods" with different name to avoid C# from picking up them